packages/code-du-travail-frontend/server.js

```diff
@@ -1,27 +1,115 @@
 // Custom server using Express.
-const express = require("express");
+const Koa = require("koa");
 const next = require("next");
-const routes = require("./routes");
+const Router = require("koa-router");
 
+const port = process.env.FRONTEND_PORT || 3000;
 const dev = process.env.NODE_ENV !== "production";
 const app = next({ dev });
-const handler = routes.getRequestHandler(app);
-
-const PORT = process.env.FRONTEND_PORT || 3000;
+const handle = app.getRequestHandler();
 
 app.prepare().then(() => {
-  express()
-    .use(handler)
-    .listen(PORT, err => {
-      if (err) throw err;
-      console.log(`
+  const server = new Koa();
+  const router = new Router();
+
+  // http://localhost:3000/a-propos
+  router.get("/a-propos", async ctx => {
```

Comment:
Should I put these routes in a routes.js file or is it ok to leave them here ? It's ok in my opinion but migh lack of clarity on why we have a server.js file